firewalld/init.sls

```diff
@@ -16,6 +16,9 @@ firewalld-unsupported:
 {% elif firewalld.enabled %}
 
 include:
+  {% if grains['osfinger'] == 'Debian-10' %}
```

Comment:
You'll see that the [Arch Linux test instance](https://gitlab.com/saltstack-formulas/firewalld-formula/-/jobs/1281244850) is failing with this error:

```sls
       local:
           Data failed to compile:
       ----------
           Rendering SLS 'base:firewalld' failed: Jinja variable 'salt.loader_context.NamedLoaderContext object' has no attribute 'osfinger'
```

That's because there is no `osfinger` grain for Arch Linux.  The way we get around this is with something like the following construct:

```suggestion
  {% if grains.get('osfinger', '') == 'Debian-10' %}
```

Reply:
Didn't know about this. Thanks for your advice.

Reply:
Or maybe even `salt["grains.get"]("osfinger", "")`?